pypesto/optimize/optimizer.py

```diff
@@ -1087,7 +1087,7 @@ class FidesOptimizer(Optimizer):
 
     def __init__(
         self,
-        hessian_update: Optional['HessianApproximation'] = 'Hybrid',
+        hessian_update: Optional['HessianApproximation'] = 'default',
```

Comment:
I would simply have `None` as default, not a string `"default"`. s.t. None means use whatever you have and approximation otherwise.

Reply:
As far as I understand fides correctly, `None` has a special meaning (i.e. handling a `None` Hessian approximation is something different as what we intend here...). Therefore we need that...